Automattic/delphin
# Suggestions: Generate a random search when users need inspiration

This adds random search terms to the search field when a users clicks 'Need some inspiration?'.

<img width="992" alt="screen shot 2016-05-18 at 5 07 45 pm" src="https://cloud.githubusercontent.com/assets/1130674/15378758/1a34da8e-1d1b-11e6-8489-01550a0de04d.png">

**Testing**
- Visit `/`
- Submit the search field with an empty query, either by hitting enter or clicking the button.
- Assert that the notice above appears below the search field.
- Click 'Need some inspiration?'.
- Assert that three random words are added to the search field.
